src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -237,6 +251,40 @@ def _reset(self):
         self.log.debug("Remote scheduler option:", self.remoteScheddOptions)
         return S_OK()
 
+    def _executeCondorCommand(self, cmd, keepTokenFile=False):
+
+        tFile = None
+        if self.token:
+            # Create a new token file if we do not keep it across several calls
+            if keepTokenFile:
+                tFile = self.tokenFile
+            if not tFile:
+                fd, tFile = tempfile.mkstemp(suffix=".token", prefix="HTCondorCE_", dir=self.workingDirectory)
```

Comment:
Do these files ever get cleaned up?

Reply:
Yes. Either in this same function if keeping tokens across several calls is not requested. Or in the calling function, e.g. killJob,  where the same token file is used for several calls.

Reply:
I'd suggest using `NamedTemporaryFile` rather than `mkstemp` in which case the file will be cleaned automatically when the object is garbage collected.

Reply:
OK, done